ibm/resource_ibm_is_instance_group_manager_action.go

```diff
@@ -470,6 +480,11 @@ func resourceIBMISInstanceGroupManagerActionDelete(d *schema.ResourceData, meta
 	deleteInstanceGroupManagerActionOptions.InstanceGroupManagerID = &instancegroupmanagerscheduledID
 	deleteInstanceGroupManagerActionOptions.ID = &instanceGroupManagerActionID
 
+	_, healthError := waitForHealthyInstanceGroup(instanceGroupID, meta, d.Timeout(schema.TimeoutUpdate))
```

Comment:
in delete use 
schema.TimeoutDelete

I think you need to define the timeout for this resource
```
Timeouts: &schema.ResourceTimeout{
			Create: schema.DefaultTimeout(10 * time.Minute),
			Update: schema.DefaultTimeout(10 * time.Minute),
			Delete: schema.DefaultTimeout(10 * time.Minute),
		},
		```